frontend/utils/time.js

```diff
@@ -0,0 +1,5 @@
+'use strict'
```

Comment:
Why explicitly saying strict? By default, I believe that our configs (and linters) are already in strict mode.

Reply:
I believe you have to have this as a line at the top of each `.js` file... sometimes even at the top of certain functions. At least JavaScript works like that to my understanding.

Reply:
Not a problem for me. It can be a useful information for some editor.

Reply:
I think it's just "noise" because babel already does that by default for us.

@pieer what editor are you talking about? As far as I know for VScode, Atom and Sublime it does nothing 🤔

Reply:
We want to follow the JS language specs explicitly if we can, and put this at the top of every file for performance and safety reasons. Plus, since we might one day remove `babel` as a dependency, that's all the more reason to do it.

Reply:
If you want to debug a single file of code online or in the browser for example. But really it doesn't matter. If you want to remove, let's remove it.